HAVE_SPLICE is probably enough, but never hurts to be safe

--- origsrc/eio-1.7.0/src/lib/eio_file.c	2012-08-30 03:52:51.000000000 -0500
+++ src/eio-1.7.0/src/lib/eio_file.c	2012-09-10 01:02:54.929036000 -0500
@@ -669,7 +669,7 @@ eio_file_copy_do(Ecore_Thread *thread, E
      }
    else
      {
-#ifdef HAVE_XATTR
+#if defined(HAVE_SPLICE) && defined(HAVE_XATTR)
        _eio_file_copy_xattr(thread, copy, in, out);
 #endif
      }
